src/content/docs/reference/Attributes/vaColor.mdx

```diff
@@ -20,4 +20,9 @@ import { Aside } from '@astrojs/starlight/components';
 
 The vertex color attribute, equivalent to `gl_Color` from the compatibility profile.
 
-The color attribute is often used to apply tints to colored blocks such as leaves, grass, water, etc. It also contains the [vanilla ao](/reference/constants/ambientocclusionlevel) and if enabled the [old lighting](/reference/shadersproperties/oldlighting). Enabling [`separateAo`](/reference/shadersproperties/separateao) will remove the vanilla ao from the rgb components of the color and put the ao in the alpha component.
\ No newline at end of file
+The color attribute is often used to apply tints to colored blocks such as leaves, grass, water, etc. It also contains the [vanilla ao](/reference/constants/ambientocclusionlevel) and, if enabled, the [old lighting](/reference/shadersproperties/oldlighting).
+
+Enabling [`separateAo`](/reference/shadersproperties/separateao) will move the AO from the `rgb` components of `vaColor` to the alpha component, which can be applied later like this:
```

Comment:
Since core profile isn't really supported, it's better to at least include compatibility profile stuff as well

Reply:
Oh sorry, I completely missed this is the docs for `vaColor`, ignore what I said above